ingest-app/src/cmr/ingest/services/granule_bulk_update_service.clj

```diff
@@ -66,11 +73,22 @@
                   (catch Exception e
                     (error "validate-and-save-bulk-granule-update caught exception:" e)
                     (let [message (.getMessage e)
-                          user-facing-message (if (string/includes? message "GBUT_PN_I")
-                                                "Granule bulk update task name needs to be unique within the provider."
-                                                message)]
+                          user-facing-message
+                          (condp #(string/includes? %2 %1) message
+                            "GBUT_PN_I"
+                            "Granule bulk update task name needs to be unique within the provider."
+
+                            "ORA-00001: unique constraint (CMR_INGEST.BUGS_PK)"
```

Comment:
This check should happen before any of the database operation takes place, i.e. we don't want a granule bulk update task created if it is invalid.

Reply:
Should the check of the name of the job being unique per provider also be done prior to insertion?